### ISSUE TITLE
[suggestion] Turret reset position Idea

### ISSUE THREAD

**Issue by Xenoprophet-Nail** [tsunamayo/Starship-EVO#3063]
Is it possible to change how turrets move to their reset point?
I was playing around with a turret and when I let go of it's controls it took the long way to get to it's reset point.
Is it possible to have the turret take the shortest path to to the reset point instead?
as you can see in the video I enclosed the guns clip through the cockpit instead of taking the easier path

### Environment
- Starship EVO version: 20w41c
- Graphics Card: Nvidia Geforce GTX 1060 6GB

### Additional information:
https://i.imgur.com/BZ2eBqT.mp4

**Comment by CurioInventorium** [comment]
I agree with this. I was trying to a rotator into position and it'll always move clockwise to the angle, no matter how close it is to it initially.
Say I'm at 0 degrees and I want it to move to 310, the turret will move clock-wise all the way to 310 instead of anti-clockwise, the shortest distance of 50 degrees.
And not being able to put -50 is frustrating too.

**Comment by tsunamayo** [comment]
Yeah I knew I still need to have a look into that, I will keep you guy posted! Cheers
